src/vm/lib/tests_helper.h

```diff
@@ -0,0 +1,31 @@
+#ifndef VM_TESTS_HELPER_H
+#define VM_TESTS_HELPER_H
+
+/**
+ * @authors Vinicius Gomes de Souza - 150047941
```

Comment:
Remove the author property of the documentation as `git blame` already do this job and the information is unnecessary in the Doxygen generated files.